app/components/FactsPageContent.tsx

```diff
@@ -17,13 +20,21 @@ const FactsPageContent = ({
   totalCount,
 }: FactsPageContent) => {
   useSyncAtom(earthquakesAtom, earthquakes);
+  const currentDate = useAtomValue(currentDateStringAtom);
 
   return (
-    <main className="flex flex-col gap-4 bg-black h-screen text-white">
-      <p>Number of quakes today: {totalCount}</p>
+    <main className="flex flex-col gap-4 bg-black-900 h-screen text-white p-10 ">
+      <h1 className="text-5xl">{currentDate}</h1>
```

Comment:
This should probably say "Overview" and then call out the date

Reply:
<img width="1201" alt="image" src="https://github.com/ash-bergs/earthquake-tracker/assets/65979049/1145f590-f7df-4164-910b-cc7807f7b365">